README.md

```diff
@@ -54,26 +54,86 @@ If you're using another operating system, you will need to ensure `mkcert` is in
 * To run the **development web app**: `npm run web`
 * Changes applied to Javascript will be applied automatically via WebPack as configured in `webpack.dev.ts`
 
-## Running the iOS app 📱
+## Running the mobile application using Rock 🪨
+
+This project uses [Rock](https://rockjs.dev/) to manage native builds. Rather than compiling native code locally when running commands like `npm run ios` or `npm run android`, Rock first attempts to download remote builds (artifacts prebuilt on CI) from a server. If a matching remote build isn’t available, it automatically falls back to building locally.
+
+By storing complete native build artifacts remotely, Rock reduces the need for local compilation and simplifies setup through automated downloads.
+
+**Note:** Any changes to files involved in generating a fingerprint (e.g., `package.json`) will trigger a local build.
+
+The following steps describe how to configure the project to fully utilize Rock.
+
+### Generating GitHub Personal Access Token
+
+To take advantage of remote builds, setup your GitHub Personal Access Token (PAT) in your `.env` file:
+
+1. Create a GitHub Personal Access Token:
+   - Go to [GitHub Settings > Developer Settings > Personal Access Tokens](https://github.com/settings/tokens)
+   - Click "Generate new token (classic)"
+   - Select the following scope:
+     - `repo`
+
+2. Copy the generated token
+
+3. Add `GITHUB_TOKEN` to `.env` file with your generated token
+
+### Running the mobile application 📱
+* To install project dependencies run: `npm install`
+* To start metro server run: `npm run start`
+**Note:** For now this is a required step — metro needs to be called manually in a separate terminal.
+* To run application on a **Development Simulator**: `npm run ios` or `npm run android`
+
+After completing these steps, you should be able to start both mobile platform apps using the remote build.
+
+### Troubleshooting
+* Try re-installing dependencies:
+    - `npm run i-standalone` for the standalone app
+    - `npm install` for the hybrid app
+
+* Try running `npm run ios` or `npm run android` again
```

Comment:
I'd mention different commands for hybrid and standalone (`ios-standalone` and `android-standalone`)